index.js

```diff
@@ -1,6 +1,37 @@
 /* eslint-env node */
 'use strict';
 
+const VersionChecker = require('ember-cli-version-checker');
+
 module.exports = {
-  name: 'ember-decorators'
+  name: 'ember-decorators',
+
+  included(app) {
+    this._super.included.apply(this, arguments);
+
+    if (!this._registeredWithBabel) {
+      let checker = new VersionChecker(this).for('ember-cli-babel', 'npm');
```

Comment:
We need to check our parents version of ember-cli-babel (not our own). I think you need to pass `this.parent` not `this`.